instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/OpenTelemetryDriver.java

```diff
@@ -99,8 +102,48 @@ public static boolean isRegistered() {
     return REGISTERED.get();
   }
 
+  /**
+   * Register underlying driver that wasn't registered against {@link DriverManager}. This is mainly
+   * useful when drivers are initialized at build time, and you don't want to postpone class
+   * initialization (f.e. runtime initialization could make driver incompatible with other systems).
+   * Drivers registered with this method has higher priority over those registered against {@link
+   * DriverManager}.
+   *
+   * @param driver {@link Driver} that should be registered
+   */
+  public static void registerDriver(Driver driver) {
```

Comment:
I would probably rename it to addDriverCandidate() or something like it. There is no guarantee that it will actually end up registered.

Reply:
Renamed, thanks.